requirements/test.in

```diff
@@ -1,11 +1,6 @@
 -r base.in
 
-flake8
-flake8_docstrings
-black
-pydocstyle~=3.0
```

Comment:
Are these changes intentional? (Looks like these have been moved into `tox.ini`. Sure, though we lose the ability to update pins)

Reply:
well, we don't really need them to run the tests...but you're right, we lose pinning by putting them inside tox.